plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1409,9 +1413,10 @@ public Optional<ConnectorOutputMetadata> finishCreateTable(
             if (handle.readVersion().isPresent()) {
                 long writeTimestamp = Instant.now().toEpochMilli();
                 DeltaLakeTableHandle deltaLakeTableHandle = (DeltaLakeTableHandle) getTableHandle(session, schemaTableName, Optional.empty(), Optional.empty());
+                TableSnapshot snapshot = getSnapshot(session, deltaLakeTableHandle);
```

Comment:
Is extracting this variable related to "Write checkpoint if necessary for CREATE OR REPLACE statements"?

Reply:
This line was outdated and not present on the last version of the file, so no need to revert - thanks for merging!

Reply:
It wasn't outdated. I reverted the change instead.